compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUOps.td

```diff
@@ -496,4 +496,26 @@ def IREEGPU_BufferResourceCastOp : Op<IREEGPU_Dialect, "buffer_resource_cast", [
   let hasCanonicalizer = 1;
 }
 
+//===----------------------------------------------------------------------===//
+// GlobalLoadDMAOp
+//===----------------------------------------------------------------------===//
+
+def IREEGPU_GlobalLoadDMAOp : Op<IREEGPU_Dialect, "global_load_dma", [
+    SameVariadicOperandSize]> {
+  let summary = "Does a global load DMA operation";
+  let description = [{
+    TODO: Add description
+  }];
+
+  let arguments = (ins AnyMemRef:$source, Variadic<Index>:$sourceIndices,
+                       AnyMemRef:$target, Variadic<Index>:$targetIndices);
```

Comment:
Different from the proposal, this op does not take tensor operands?

Is there a plan to replace `linalg.copy {lowering_config = #iree_gpu.use_global_load_dma}` with this op on tensors?

> Tagged linalg.copy will not be decomposed/tiled until bufferization.

If we use it on tensors, it seems like it naturally prevents the decomposition and tiling for the case.

Reply:
Yeah, maybe this should be DPS-y